### DISCUSSION TITLE
real-time schedule

### DISCUSSION THREAD

**Discussion by ArtifactyNight** [Zrips/CMI#2272] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
yeah, can setting for automatic restart server every day in 3 AM, time reward.. i mean every 3 PM every player online in server can get command, items etc.
### Basic explanation as to why this feature is needed and/or useful to have: ###
don't use any AutoRestart,RealTime  Schedule Plugin

**Comment by mercurialmusic** [comment]
If your control panel doesn't already support this, it is possible in CMI's schedules.yml, including using real time. Everything is explained in that file, and it's quite easy to use...
